src/io/flutter/actions/OpenAndroidModule.java

```diff
@@ -61,8 +61,8 @@ private static void openOrImportProject(@NotNull VirtualFile projectFile,
     if (canImportAsGradleProject(projectFile)) {
       VirtualFile target = findGradleTarget(projectFile);
       if (target != null) {
-        GradleProjectImporter gradleImporter = GradleProjectImporter.getInstance();
-        gradleImporter.importAndOpenProjectCore(null, true, projectFile);
+        com.intellij.ide.impl.ProjectUtil.openOrImport(projectFile.toNioPath(), OpenProjectTask.build().withForceOpenInNewFrame(forceOpenInNewFrame));
```

Comment:
I'd be inclined to replace this qualified name with an `import` statement? Or maybe there's some reason not to?

Reply:
After looking into this more, I think my solution is wrong because this method `openOrImport` is already being imported statically and is used below. There's some additional complexity here about whether a directory should be opened as though it's a project in itself or as a sub-project, but I'm not sure what all of the consequences are. I think I'll mark this PR a draft for now.